src/fsharp/AttributeChecking.fs

```diff
@@ -267,9 +267,16 @@ let private CheckILAttributes (g: TcGlobals) isByrefLikeTyconRef cattrs m =
 
 /// Check F# attributes for 'ObsoleteAttribute', 'CompilerMessageAttribute' and 'ExperimentalAttribute',
 /// returning errors and warnings as data
-let CheckFSharpAttributes g attribs m = 
-    if isNil attribs then CompleteD 
-    else 
+let langVersionPrefix = "--langversion:preview"
```

Comment:
The comment above this value is not attached to the `CheckFSharpAttributes` anymore.